### PR TITLE
initial index.html commit

### PR DESCRIPTION
**Pull request by Rhowie86**
## changes

Initial commit for index.html

## testing

Just fetch and make sure it looks good

### CODE REVIEW

**Review by gthompson2 — APPROVED** [review]
👍🏾

File: index.html [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+    <main class="container">
+        
+        <header class="header-container">
+            <div class="logo-container"></div>
+            <div class="navBar-container">
+
+            </div>
+
+        </header>
+
```
Comment (cslunetta) [inline index.html]: should we add an article or div for the dropdowns to live in? 

`<article class="locationSelect-dropdowns"`

**Review by cslunetta — COMMENTED** [review]
should we add in a spot after the header for the dropdowns?

**Review by cslunetta — APPROVED** [review]
Looks good!
👍🏻

**Review by gthompson2 — APPROVED** [review]

**Pull request merged**